templates/bundles/TwigBundle/Exception/error500.html.twig

```diff
@@ -0,0 +1,25 @@
+{% extends '@W3CWebsiteTemplates/bundles/TwigBundle/Exception/error.html.twig' %}
+
+{% block documenttitle %}{{ 'error.500.title'|trans({}, 'w3c_website_templates_bundle') }} | W3C{% endblock documenttitle %}
+
+{% block title %}{{ 'error.500.title'|trans({}, 'w3c_website_templates_bundle') }}{% endblock title %}
+
+{% block content %}
+    {% embed '@W3CWebsiteTemplates/components/text.html.twig' %}
+        {% block text %}
+            {{ 'error.500.message'|trans({}, 'w3c_website_templates_bundle')|raw }}
+        {% endblock %}
+    {% endembed %}
+
+    {% if is_granted('ROLE_ADMIN') and exception %}
+        {% embed '@W3CWebsiteTemplates/components/text.html.twig' %}
+            {% block text %}
+                <h2>Error details</h2>
+                <p>Message: <strong>{{ exception.message }}</strong></p>
+                <p>in <em>{{ exception.file }}: {{ exception.line }}</em></p>
```

Comment:
Let's also use the `file_relative` filter here.

```suggestion
                <p>in <em>{{ exception.file|file_relative }}: {{ exception.line }}</em></p>
```

Reply:
I didn't use that filter here as it'll be only be displayed to admin. Also the stacktrace will also include the full path to the different files so it won't make much difference.

Reply:
Yes my comment was not related to security but more for readability and having a shorting string on that message.

Reply:
ok, sounds good.